pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTStage.java

```diff
@@ -102,31 +116,37 @@ public String toGroovy() {
         if (environment != null) {
             result.append(environment.toGroovy());
         }
-        result.append("steps {\n");
-        if (branches.size() > 1) {
-            result.append("parallel(");
-            boolean first = true;
-            for (ModelASTBranch branch: branches) {
-                if (first) {
-                    first = false;
-                } else {
-                    result.append(',');
+        if (branches.isEmpty() && parallelStages != null) {
+            result.append("parallelStages {\n");
```

Comment:
`parallelStages` seems reasonable to me.    But we it is probably worth getting buy off from more people (and updating the RFC?).  Questions like: can `parallelStages` be used instead of `stages` at the top of pipeline?  

The first thing someone is going to do when they see `stage('A') { parallelStages { stage ('B') ...` is expect to be able to do `stage('A') { parallelStages { stage ('B') { agent { ...`.  And that is not supported in this implementation, right? 

I'd almost rather push this out and do this in a way that is consistent with the `stage` behavior elsewhere.